Description: Don't build non-DFSG (GFDL 1.1 licensed) help files
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
--- a/Makefile.am
+++ b/Makefile.am
@@ -35,8 +35,7 @@
 	autogen.sh \
 	omf.make \
 	xmldocs.make \
-	COPYING.libs \
-	COPYING.docs
+	COPYING.libs
 
 DISTCLEANFILES =                \
 	gtk-doc.make
--- a/baobab/Makefile.am
+++ b/baobab/Makefile.am
@@ -1 +1 @@
-SUBDIRS = src data pixmaps help
+SUBDIRS = src data pixmaps
--- a/configure.ac
+++ b/configure.ac
@@ -385,16 +385,13 @@
 baobab/pixmaps/24x24/Makefile
 baobab/pixmaps/scalable/Makefile
 baobab/src/Makefile
-baobab/help/Makefile
 
 logview/Makefile
 logview/data/Makefile
 logview/tests/Makefile
-logview/help/Makefile
 
 gsearchtool/Makefile
 gsearchtool/data/Makefile
-gsearchtool/help/Makefile
 gsearchtool/libmateui-deprecated/Makefile
 
 mate-dictionary/Makefile
@@ -406,7 +403,6 @@
 mate-dictionary/docs/reference/Makefile
 mate-dictionary/docs/reference/gdict/Makefile
 mate-dictionary/docs/reference/gdict/version.xml
-mate-dictionary/help/Makefile
 mate-dictionary/src/Makefile
 
 mate-screenshot/Makefile
--- a/gsearchtool/Makefile.am
+++ b/gsearchtool/Makefile.am
@@ -1,6 +1,6 @@
 Utilitiesdir = $(datadir)/applications
 
-SUBDIRS = data help libmateui-deprecated
+SUBDIRS = data libmateui-deprecated
 
 AM_CPPFLAGS = \
 	-I$(srcdir)/libmateui-deprecated \
--- a/logview/Makefile.am
+++ b/logview/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = data help tests
+SUBDIRS = data tests
 
 AM_CPPFLAGS = \
 	-DG_LOG_DOMAIN=\"mate-system-log\"			\
--- a/mate-dictionary/Makefile.am
+++ b/mate-dictionary/Makefile.am
@@ -1,3 +1,3 @@
-SUBDIRS = libgdict src data docs help
+SUBDIRS = libgdict src data docs
 
 EXTRA_DIST = TODO README AUTHORS
